### DISCUSSION TITLE
iife as module output type missing?

### DISCUSSION THREAD

**Discussion by spocke** [swc-project/swc#2720] [Q&A]
I experimented with swc and from what I understand there is no support for outputting the bundle as a iife. So either it's polluting the global namespace or tries to act like a module to something else.

**Comment by kdy1** [comment]
There's one in rust api, but not exposed to javascript.
I'm planning a large refactor of bundler api, and I'll add it, too.

**Reply by spocke** [reply]
Ok sound nice. I saw that this ticket was closed https://github.com/swc-project/swc/issues/1184 but couldn't find a reference to any iife config option.

**Reply by Vadorequest** [reply]
I would also be interested in this feature!

**Reply by DrSensor** [reply]
is it available?
looking at the docs, there is no option for IIFE in `module.type`
